style.css

```diff
@@ -385,3 +388,92 @@ header button {
 header button:hover {
   background-color: #ca5400af;
 }
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 120px;
+}
+
+.contact-form h2 {
+  width: 327px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+}
+
+.contact-form form {
+  padding-top: 50px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #979493;
+}
+
+.contact-form form input {
+  border: none;
+  width: 327px;
+  border-bottom: 1px solid #dbd8d7;
+  border-radius: 4px;
+  padding: 14px;
+}
+
+.contact-form form textarea {
+  background: #fbf8f7;
+  border: none;
+  border-radius: 4px;
+  padding: 14px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #3c3a39;
+}
```

Comment:
- [ ] Kindly make sure your `textarea` looks as close to the Figma design as possible 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/177412163-86ee6a55-69a3-48f2-9f73-da19c1d76c68.png)  |  ![image](https://user-images.githubusercontent.com/74814780/177412190-8ed7d7c8-3293-4d5e-9b9b-1b97faac8cc2.png)